README.md

```diff
@@ -104,10 +104,75 @@ To get specific instructions for your preferred shell run:
 mongocli completion <bash|zsh|fish|powershell> --help
 ```
 
+## AtlasCLI (Pre-Release)
+![GO tests](https://github.com/mongodb/mongocli/workflows/GO%20tests/badge.svg)
+![golangci-lint](https://github.com/mongodb/mongocli/workflows/golangci-lint/badge.svg)
+
+The MongoDB AtlasCLI is a modern command line interface that enables you to manage MongoDB Atlas from the terminal.
+
+![atlascli-atlas-quickstart](https://user-images.githubusercontent.com/5663078/156184669-57c8ddce-6f0a-4e84-9311-2d996cb27942.gif)
+
+## Installing
+
+AtlasCLI is currently in the pre-release phase, so it should not be used in production environment. 
+
+### Pre-built Binaries
+
+Download the appropriate version for your platform from [AtlasCLI releases](https://github.com/mongodb/mongocli/releases).
```

Comment:
[q] once we rename the repo, this link will display mongo and atlascli releases in the same link?

Reply:
yes, correct

Reply:
A follow-up here, will the GitHub APIs to list releases work on the previous repo URL?

Reply:
meaning the CLIs out there already (v1.20.0 for instance) will that one be able to verify latest version after repo rename?

Reply:
This comment is not meant to block this PR btw